plans/integration/remove-all-submgr-pkgs/main.fmf

```diff
@@ -47,4 +47,4 @@ provision:
     - name: install rhsm
       how: ansible
       playbook:
-        - tests/integration/remove_all_submgr_pkgs/ansible/install_submgr_8.yml
+        - tests/ansible_collections/roles/install-submgr/install_submgr_centos_8.yml
```

Comment:
I saw tests/ansible_collections/roles as the roles for the initial preparation of the testing environment. However, I was adding other playbooks in the root of tests/ansible_collections (to be consumed by some specific tests only).

How do you see this tests/ansible_collections/roles/install-submgr/install_submgr_centos_8.yml will be used? Maybe we can create a different directory tree?

Reply:
I considered the ansible_collections as a folder for also roles used by multiple tests, to avoid duplication. At this moment I don't have a problem with the current dir structure. Let me know if you have an idea on how to restructure the tree.

Reply:
I saw it the same. But I'd do not mix the roles for initial preparation and roles for tests. This will allow us to easy share it then with the leapp team for example